### PR TITLE
fix(dedicated-server): ip monitoring management

### PR DESCRIPTION
**Pull request by radireddy**
not able to add, edit and delete ips for monitoring

closes #DTRSD-19171

| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-19171
| License          | BSD 3-Clause

## Description

API accepts ip address to create new ip monitoring. Manager was sending an array of ip's. I have fixed this issue. Also made a few more changes to fix edit and delete.

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report installation#14922.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#14922.1 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#14922.2 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#14922.3 ✘  
* Stage 1
  * Install ✘

File: packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/monitoring.html [hunk 1]
```diff
@@ -464,7 +464,7 @@ <h2 data-translate="server_tab_MONITORING_services"></h2>
         >
             <!-- monitoring services -->
             <tr
-                data-ng-repeat="monitoring in $ctrl.monitoringServices track by monitoring.monitoringId"
+                data-ng-repeat="monitoring in $ctrl.monitoringServices track by $index"
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/monitoring.html:467]: `monitoring.monitoringId` may not be unique ?

Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/monitoring.html:467]: no not when there are 2 monitoring entries for same IP

**Review by JeremyDec — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Pull request merged**